src/pages/Experiments/TasksMenuBlock/TasksMenu/style.less

```diff
@@ -9,6 +9,10 @@
 
   padding-left: 24px !important;
 
+  .iconUrl {
```

Comment:
Class names should be written in **kebab case**, example: `icon-url`. If you find a class name in a different format, feel free to change it.